mage_ai/frontend/utils/date.ts

```diff
@@ -135,6 +150,40 @@ export function utcNowDate(opts?: { dateObj?: boolean }): any {
   return utcDate;
 }
 
+// Return a map of the current time in the different provided timezones
+export function currentTimes({ 
+  timeZones, 
+  includeSeconds = false,
+}: { 
+  timeZones: TimeZoneEnum[];
+  includeSeconds?: boolean;
+}) {
+  const currentMoment = tzMoment.utc();
+  const zoneTimes = new Map(
+    timeZones.map((timeZone) => {
+      let moment = currentMoment;
+      switch (timeZone) {
+        case TimeZoneEnum.LOCAL:
+          moment = currentMoment.local();
+        case TimeZoneEnum.UTC:
```

Comment:
Is this case clause still needed since it is empty?

Reply:
It's not technically needed but I just left it there for increased clarity that UTC is the default in the case that we add a lot more time zones here!